static/app/views/performance/data.tsx

```diff
@@ -456,6 +457,10 @@ function generateGenericPerformanceEventView(
     eventView.additionalConditions.addFilterValues('event.type', ['transaction']);
   }
 
+  if (isTrends) {
+    eventView.additionalConditions.addFilterValues('has', ['measurements.vital']);
+  }
```

Comment:
`has:measurements.vital` should only be added to the trends query (that was the assumption i made)

Reply:
`measurements.vitals` is not an actual measurement. In this case, the measurement we want to add depends on the parameter dropdown. So if `LCP` was selected, you'll want to add `has:measurements.lcp` and so on. Try doing this manually in the search bar to see how that affects the results.

![image](https://user-images.githubusercontent.com/10239353/149386167-48038d31-8de1-445e-b06b-d92960013c0b.png)

Reply:
sorry my initial understanding of the task wasn't accurate. Updated the approach